cmake/toolchains/riscv64-100-thead-gnu.toolchain.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# NOTE: use T-Head compiler:
+#    git clone https://github.com/T-head-Semi/xuantie-gnu-toolchain.git
```

Comment:
Recently the group `T-Head-Semi` has been renamed to `Xuantie-RV`.
Could you please replace this repository link with [new address](https://github.com/XUANTIE-RV/xuantie-gnu-toolchain)? Also correct please in the toolchain file for rvv0p7.

Reply:
Sure! I will replace these addresses!